tests/ManageCourses.Tests/UnitTesting/PublishServiceTests.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System.Collections.Generic;
+using FluentAssertions;
+using GovUk.Education.ManageCourses.Api.Data;
+using GovUk.Education.ManageCourses.Api.Model;
+using GovUk.Education.ManageCourses.Api.Services;
+using GovUk.Education.ManageCourses.Api.Services.Publish;
+using GovUk.Education.ManageCourses.Domain.Models;
+using GovUk.Education.SearchAndCompare.Domain.Client;
+using Moq;
+using NUnit.Framework;
+
+namespace GovUk.Education.ManageCourses.Tests.UnitTesting
+{
+    [TestFixture]
+    public class PublishServiceTests
```

Comment:
The happy path test is too thin and too mocked out

ie 
searchAndCompareApiMock.Setup(x => x.SaveCoursesAsync(It.IsAny<List<SearchAndCompare.Domain.Models.Course>>())).ReturnsAsync(true);

Not even checking how may items to be sent.